haulage.sql

```diff
@@ -16,13 +16,13 @@ CREATE TABLE `customers` (
   `admin` tinyint(1) DEFAULT '0',
   `msisdn` varchar(16) NOT NULL,
   PRIMARY KEY (`imsi`)
-) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
 
 CREATE TABLE `static_ips` (
   `imsi` varchar(16) NOT NULL,
   `ip` varchar(16) NOT NULL,
   PRIMARY KEY (`imsi`)
-) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
```

Comment:
I can't figure out why we need this CHARSET change? If it's unrelated and just better practice, that's fine, but if it's vital for operation I will have to manually make sure our in the wild DBs are migrated...

Reply:
It's not vital for operation. With latin-1 there are characters (like emoji) that will corrupt the database strings if input as usernames. Making this change for new databases should not impact existing deployed databases.